src/ensembl/src/content/app/species/components/species-title-area/SpeciesTitleArea.scss

```diff
@@ -0,0 +1,91 @@
+@import 'src/styles/common';
+
+$container-padding-right: 20px;
+
+@mixin narrow-grid {
+  grid-template-areas:
+  'icon species-name species-name species-name'
+  'icon use-dont-use . remove';
```

Comment:
Having this `.` here creates a `60px` gap between `use-dont-use` & `remove`. Is this okay?

Reply:
Yes, that's exactly why the `.` is there :-) Andrea wanted a 60px gap, and css grid can't define widths for specific gaps.